pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -84,6 +86,7 @@
   private Map<String, ForwardIndexCreator> _forwardIndexCreatorMap = new HashMap<>();
   private Map<String, DictionaryBasedInvertedIndexCreator> _invertedIndexCreatorMap = new HashMap<>();
   private Map<String, DictionaryBasedInvertedIndexCreator> _textIndexCreatorMap = new HashMap<>();
+  private Map<String, DictionaryBasedInvertedIndexCreator> _fstIndexCreatorMap = new HashMap<>();
```

Comment:
You should rebase. I think these interfaces are no longer there. 
Also, we should not base this off InvertedIndexReader and InvertedIndexCreator interface. Now we have separate interface for TextIndexReader and TextIndexCreator. It was recently decoupled from InvertedIndex interface